### PR TITLE
Update beijing.geojson

### PR DESCRIPTION
**Pull request by yozman**
- 添加 Costa Coffee（奥林匹克公园地铁站 D出口）

### CODE REVIEW

**Comment by sinchang** [comment]
please update store number in README.md also. @yozman

**Comment by yozman** [comment]
@sinchang
already done.

File: beijing.geojson [hunk 1]
```diff
@@ -1395,6 +1395,24 @@
...
         ]
       }
+    },
+    {
+      "type": "Feature",
+      "properties": {
+        "名称": "Costa Coffee（奥林匹克公园地铁站 D出口）",
+        "下载速度": "2.81 Mbps",
+        "Speedtest 链接": "http://www.speedtest.net/result/6254188263",
+        "评论 (@xhacker)": "环境舒适、比较吵，靠边处有少量电源。",
```
Comment (xhacker) [inline beijing.geojson]: Can you update `@xhacker` to you name? :)

**Comment by yozman** [comment]
@xhacker 
done

**Comment by xhacker** [comment]
Thanks!

**Pull request merged**